index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Intuit Accounts - Sign Up</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <header class="login-wrapper">
+      <div class="login-navigation">
+        <div class="forgot-column">
+          <div id="forgot-text">Already have an Intuit account?</div>
+          <a href="#" id="recovery-link">I forgot my user ID or Password</a>
+        </div>
+        <div class="login-column">
+          <button id="login" class="login-submit" type="button">
+            <span class="lock-img"></span>
+            Sign In
+          </button>
+        </div>
+      </div>
+    </header>
+    <header class="main-header" id="main-header">
+      <div class="logo-container">
+        <a href="#" id="main-logo" class="logo-brand"></a>
+      </div>
+    </header>
+    <section class="form-main">
+      <div class="sub-header">
+        <div class="logo-outside-card">
+          <header class="mini-main-header">
+            <div class="mini-logo-container">
+              <ul class="logo-list">
+                <li class="mint-icon"></li>
+                <li class="mint-text"></li>
+                <li class="qbooks-icon"></li>
+                <li class="qbooks-text"></li>
+                <li class="ttax-icon"></li>
+                <li class="ttax-text"></li>
+              </ul>
+            </div>
+          </header>
+        </div>
+      </div>
+      <section class="wrapper">
+        <div class="title">
+          <span class="title-head">Create an Intuit account</span>
+        </div>
+        <p class="text-title">
+          One account for everything Intuit, including Mint.
+        </p>
+        <p class="text-link">Learn more</p>
```

Comment:
- [ ] **[MANDATORY]** I see you gave this text some stylings to simulate a link, however, you should use the correct HTML tag, that is the anchor link tag `<a href="#">` Please fix it in all the text that is supposed to be a link. Strangely you did it right on the links outside of the form

Reply:
Done.